zipkin-lens/README.md

```diff
@@ -9,37 +9,33 @@ Here are a couple example screen shots:
 
 ## Quick start
 
-Zipkin-lens required nodejs 10.2.0 to start. You could easily try Zipkin-lens at local using any zipkin deployment using below command:
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
maybe revert this stuff?

Reply:
I left it in as a shout-out to the open source project. I removed the Learn More section since it seems extraneous but am leaning towards leaving this section, let me know if that makes sense.

Reply:
personally I think this dismisses the actual work, many many hours by several people.

anyway, there was a lot of other stuff removed from the README, was that intended?

Reply:
Ok removed the link. And completely brain-farted on the rest of the README, copied it back in.